draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1901,9 +1904,66 @@ and feedback.
 All references to {{RFC4122}} in the IANA registries should be replaced with references to this document.
 References to {{RFC4122}} document's Section 4.1.2 should be updated to refer to this document's {{format}}.
 
-There is no update required to the IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}}.
-
-IANA is not required to track UUIDs used for identifying items such as versions, variants, namespaces, or hashspaces.
+The IANA URN namespace registration {{URNNamespaces}} for UUID filed in {{RFC4122}} should be updated to reference this document.
+
+Finally IANA should track UUID Variants, UUID Subtypes, and Special Case "Reserved UUIDs" such as Min, Max, and Namespace IDs as specified in {{iana1}}, {{iana2}}, {{iana3}}.
+
+The policy for these three registries is "Specification Required", per {{RFC8126}}, Section 4.6.
```

Comment:
I was leaning that way at first but UUIDv2 is a non-RFC and some types like the MS one, or even the 111 var and whatever subtype could be claimed by another standard body. 
Thus I relaxed it since UUIDs are not just in the IETF space.